src/applications/vaos/sass/vaos.scss

```diff
@@ -42,3 +42,13 @@
     margin-top: 0.6em;
   }
 }
+
+.vaos-form__audiology-type {
```

Comment:
Could we combine this with the `.vaos-form__appt-type` styles? It seems like the same kind of UI.